.gitignore

```diff
@@ -120,3 +120,5 @@ app.*.symbols
 !**/ios/**/default.perspectivev3
 !/packages/flutter_tools/test/data/dart_dependencies_test/**/.packages
 !/dev/ci/**/Gemfile.lock
+/packages/flutter/example/linux/flutter/ephemeral/
```

Comment:
I don't think this is needed; if you rebase on master, there is already an entry in the root `.gitignore`:

```
# Linux
**/linux/flutter/ephemeral
```